src/foraging_gui/Foraging.py

```diff
@@ -2987,7 +2987,7 @@ def _Open(self,open_last = False,input_file = ''):
                 # list of mice, experimenters, and two week in chronological order form date modified
                 mice, experimenters, two_week = self._Open_getListOfMice()
                 # only add mice from two weeks in drop down.
-                W = MouseSelectorDialog(self, [m + ' ' + n for m, n in zip(two_week, experimenters)])
```

Comment:
Just for clarity, if you want to use zip, you could use
```
[m[0] + ' ' + m[1] for m in zip(two_week, experimenters)]
```

Reply:
I changed it because I thought there was a bug with the sorting by date but it was another bug of when I was populating the two_week list